data_management/access_copy_converter/bulk_access_copy_converter/main.py

```diff
@@ -0,0 +1,197 @@
+import json
+import logging
+import os
+import subprocess  # nosec
+import tempfile
+
+import boto3
+from botocore.exceptions import ClientError
+from sqlalchemy import MetaData, Table, create_engine, select
+from sqlalchemy.exc import SQLAlchemyError
+
+logging.basicConfig(level=logging.INFO, format="%(levelname)s: %(message)s")
+logger = logging.getLogger(__name__)
+
+s3 = boto3.client("s3")
+sm = boto3.client("secretsmanager")
+
+
+def get_secret_string(secret_id):
+    secret_value = sm.get_secret_value(SecretId=secret_id)
+    return json.loads(secret_value["SecretString"])  # pragma: allowlist secret
+
+
+def get_engine():
+    db_secret_id = os.getenv("DB_SECRET_ID")
+    creds = get_secret_string(db_secret_id)
+    url = (
+        f"postgresql+psycopg2://{creds['username']}:{creds['password']}"
+        f"@{creds['proxy']}:{creds['port']}/{creds['dbname']}"
+    )
+    return create_engine(url)
+
+
+def already_converted(bucket, key):
+    try:
+        s3.head_object(Bucket=bucket, Key=key)
+        return True
+    except ClientError as e:
+        if e.response["Error"]["Code"] == "404":
+            return False
+        raise
+
+
+def get_extension(file_id, conn, metadata):
```

Comment:
isn't a lot of this duplicate work from `data_management/access_copy_converter/access_copy_converter/main.py`? Can we try to abstract it out and reuse a shared function?

Reply:
yeah was rushing this a little bit 
Wanted to refactor and both functions and create a shared folder for both functions 
we could create a separate refactor ticket as the focus is on excel conversion now